transformers/integration_tests/deny_test.go

```diff
@@ -15,58 +15,48 @@ import (
 )
 
 var _ = Describe("Deny transformer", func() {
-	const (
-		defaultOffset int = 0
-		vatOffset     int = -1
-	)
-
 	Context("Cat deny events", func() {
 		usrAddress := "0x13141b8a5e4a82ebc6b636849dd6a515185d6236"
-		denyIntegrationTest(int64(14764643), test_data.CatAddress(), usrAddress, defaultOffset)
+		denyIntegrationTest(int64(14764643), test_data.CatAddress(), usrAddress, usrAddress)
```

Comment:
Kind of weird that for all these integration tests I'm passing in `usrAddress` twice, but it's because I just happened to pick `deny` events where `msg.sender`s are passing their own addresses in as the `usr` parameter. I didn't want to abstract `usrAddressHex` and `msgSenderAddressHex` (on line 59) into a single parameter, because I was worried that would imply that those 2 addresses _have to_ be the same, which they don't.

Reply:
👍